grakn-dist/src/dist.xml

```diff
@@ -28,82 +28,46 @@
-                <include>LICENSE.txt</include>
-                <include>README.md</include>
+                <include>grakn</include>
+                <include>graql</include>
             </includes>
         </fileSet>
+        <!-- changelog license readme -->
         <fileSet>
-            <outputDirectory>src</outputDirectory>
-            <directory>${main.basedir}/</directory>
+            <outputDirectory>/</outputDirectory>
+            <directory>${main.basedir}</directory>
             <includes>
-                <include>pom.xml</include>
                 <include>LICENSE.txt</include>
-                <include>grakn-core/**</include>
-                <include>grakn-client/**</include>
-                <include>grakn-test-profiles/**</include>
-                <include>grakn-dashboard/**</include>
-                <include>grakn-dist/**</include>
-                <include>grakn-engine/**</include>
-                <include>grakn-kb/**</include>
-                <include>grakn-graql/**</include>
-                <include>grakn-graql-shell/**</include>
-                <include>grakn-migration/**</include>
-                <include>grakn-test/**</include>
-                <include>grakn-factory/**</include>
+                <include>README.md</include>
             </includes>
-            <excludes>
-                <exclude>**/target/</exclude>
-                <exclude>grakn-dashboard/node_modules/</exclude>
-                <exclude>grakn-dashboard/static/dashboard.js</exclude>
-            </excludes>
         </fileSet>
         <fileSet>
-            <outputDirectory>docs/api</outputDirectory>
-            <directory>${main.basedir}/target/apidocs/</directory>
+            <outputDirectory>services</outputDirectory>
+            <directory>${basedir}/src/services/</directory>
             <includes>
                 <include>**</include>
             </includes>
         </fileSet>
-        <fileSet>
-            <outputDirectory>docs/rest-api</outputDirectory>
-            <directory>${main.basedir}/grakn-engine/target/jaxrs-raml/</directory>
-            <includes>
-                <include>engine-apis.html</include>
-            </includes>
-        </fileSet>
-        <fileSet>
-            <outputDirectory>bin</outputDirectory>
-            <directory>${basedir}/src/bin/</directory>
-            <includes>
-                <include>*</include>
-            </includes>
-        </fileSet>
+        <!-- grakn.properties -->
         <fileSet>
             <outputDirectory>conf</outputDirectory>
-            <directory>${basedir}/src/conf/</directory>
+            <directory>${main.basedir}/conf/main</directory>
             <includes>
-                <include>**</include>
+                <include>grakn.properties</include>
             </includes>
         </fileSet>
+        <!-- and logback.xml (main config) -->
         <fileSet>
-            <outputDirectory>examples</outputDirectory>
```

Comment:
I think we want to still keep the examples in the dist for the quickstart. @mikonapoli @fppt

Reply:
IMHO, people would normally just go to the website or google for a tutorial.

Reply:
I find these really helpful for quickly checking stuff works, like `graql.sh -f examples/pokemon.gql`. I don't know how useful they are to users though.

Reply:
I use them personally that way too and would say that it removes the additional step of downloading the examples for new users.

Reply:
Cool. so, to include, or not to include?

Current standing:
- don't include: Ganesh
- include: Felix, Thanh

Reply:
I fall on the include side here. I find this useful for testing and with a little more jiggering it can be made useful for users as well. Kind of how the first thing most people learning Tinkerpop do is load their embedded example. +1 Include.

Reply:
I'm for including them, we can reference them in the tutorials/docs and we know people already have them

Reply:
OK, I'll put it back in if it's useful